### DISCUSSION TITLE
The repo is inactive again?

### DISCUSSION THREAD

**Discussion by Varorbc** [microsoft/TemplateStudio#4610] [General]
The repo is inactive again?
@mikebattista @mrlacey

**Comment by mikebattista** [comment]
We caught up with the last wave of changes we made. What features in particular are you looking for?

I do plan to update to latest WinAppSDK/WinUIEx releases soon. See #4611.

**Reply by Mysterious-Dev** [reply]
So now that everything is updated, can we imagine a release soon?

**Reply by mikebattista** [reply]
There will be an update soon but we're looking at what other smaller fixes we might want to get in first.
